scan/scan.bash

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Update the git submodule(s) containing wpscan (and potentially other/future projects) so that we always run the latest version
+cd "$(dirname $0)/.."
+git submodule foreach git pull origin master
+
+# wpscan requires it's dependencies to be in PATH, easiest to just cd to folder
+cd "scan/wpscan"
+# Set wpscan script and sites to scan
+wpscan=./wpscan.rb
+sites=$(ls /etc/nginx/sites-enabled)
```

Comment:
Because it's the way wp-many-site works we can be sure that this line works for our purposes. And that's the only half-wonky thing I can find in this PR.